components/lib/tagsSelector.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react'
+import { Label, Select, TextField } from '@navikt/ds-react'
+import {
+  useKeywordsQuery,
+  useTeamkatalogenQuery,
+} from '../../lib/schema/graphql'
+import ErrorMessage from './error'
+import LoaderSpinner from './spinner'
+import { Dispatch, SetStateAction, useContext, useEffect } from 'react'
+import { Dropdown } from '@navikt/ds-react-internal'
+import CreatableSelect from 'react-select/creatable'
+import { CONTINUOUS_DOMAIN_SCALES } from 'vega-lite/build/src/scale'
+import classNames from 'classnames'
```

Comment:
this import isn't used as far as i can tell

Reply:
true!